src/applications/mhv-secure-messaging/tests/e2e/sort-filter-test/secure-messaging-advanced-filter-inbox.cypress.spec.js

```diff
@@ -48,6 +48,10 @@ describe('SM INBOX ADVANCED FIXED DATE RANGE SEARCH', () => {
 
     PatientSearchPage.verifySearchResponseLength(searchResultResponse);
     PatientSearchPage.verifyMessageDate(3);
+    PatientSearchPage.verifySearchMessageLabel(
+      searchResultResponse,
+      'Last 3 months',
```

Comment:
Looks like you repeatedly use the same string values for validations. Would it be more convenient to turn int to a constant and use a variable across?